logging/logging-loki-ocp.adoc

```diff
@@ -0,0 +1,132 @@
+
+In {logging} documentation, *LokiStack* refers to the {logging} supported combination of Loki, and web proxy with {product-title} authentication integration. LokiStack's proxy uses {product-title} authentication to enforce multi-tenancy. *Loki* refers to the log store as either the individual component or an external store. You can query Loki link:https://grafana.com/docs/loki/latest/[using JSON paths or regular expressions].
+
+include::modules/logging-support-considerations.adoc[leveloffset=+1]
+
+[id="loki-deployment-sizing_{context}"]
+== Deployment Sizing
+
+include::snippets/logging-loki-size-snip.adoc[]
+
+include::modules/logging-loki-gui-install.adoc[leveloffset=+]
+
+include::modules/logging-loki-cli-install.adoc[leveloffset=+1]
+
+=== Troubleshooting installation issues
+
+include::snippets/logging-install-trouble-snip.adoc[]
+
+== Creating a secret for your log store
+
+include::modules/logging-loki-storage.adoc[leveloffset=+1,lines=9..278]
+
+== Creating the `LokiStack` CR
+
+1. Click on *Operators* and then *Installed Operators*.
+
+  a. Select the *Loki Operator*.
+
+  b. On the first page under *Provided APIs* and *LokiStack*, select *Create instance*.
+
+  c. Switch to the *YAML view* option.
+
+  d. Replace the YAML present using the template below:
+
+include::snippets/logging-lokistack-cr-snip.adoc[]
+
+  e. Then click *Create*.
+
+== Creating the *ClusterLogging* CR instance
+
+1. Click on *Operators* and then *Installed Operators*.
+
+a. Select the *Red Hat OpenShift Logging* Operator.
+
+b. On the first page, under *Provided APIs* and *Cluster Logging*, select *Create instance*.
+
+c. Replace the YAML present using the template below:
+
+
+include::snippets/logging-clusterlogging-cr-snip.adoc[]
+
+This creates an instance of `ClusterLogging` within the namespace `openshift-logging`.
+
+d. Click *Create*.
+
+=== Verify the Logging install
+
+Now that Logging has been created, verify the installation.
+
+1. Switch to the *Workloads* → *Pods* page.
+
+2. Select the *openshift-logging* project.
+
+3. Verify that you see the following pods: *cluster logging*, *collectors*, *logging-view-plugin*, and various *lokistack* pods.
+
+4. Shortly after installation, a box in the top right corner displays "Web console update is available" and prompts you to refresh your browser. Refresh the page and verify that you see the logs.
+
+
+== Observing the logs
+
+1. Select *Observe* -> *Logs* on the left menu.
+
+2. Select *Infrastructure*.
+
+3. Verify that you see the *Infrastructure* logs grouped in *application*, *infrastructure*, and *audits* sections.
```

Comment:
This does not make sense to me. "Infrastructure" is one of our "log types" ("tenants"?). It does not include "application" and "audit" but is on the same "level" as those.

Reply:
@libander I think this is not updated yet.